### PR TITLE
Fix links to overriden methods in docs for base class

### LINKED ISSUE #1994: Hyperlinks to overridden members don't work
This page:
https://master-api.flutter.dev/flutter/rendering/Constraints-class.html
...contains a link to `BoxConstraints.debugAssertIsValid` but it actually leads to the superclass `Constraints.debugAssertIsValid` instead.

### PR DESCRIPTION
**Pull request by keertip**
Fixes #1994

### CODE REVIEW

File: test/model_test.dart [hunk 1]
```diff
@@ -1432,6 +1435,15 @@ void main() {
             contains('<a href="ex/B-class.html">ex.B</a>'));
       });
 
+      test('link to override methoed in implemetor from base class', () {
```
Comment (kevmoo) [inline test/model_test.dart]: `method`

File: testing/test_package/lib/override_class.dart [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import 'base_class.dart';
+
+class BoxConstraints extends Constraints {
+  /// Creates box constraints with the given constraints.
+  const BoxConstraints(
```
Comment (kevmoo) [inline testing/test_package/lib/override_class.dart]: weird formatting here. Shouldn't this end up being `();`?

File: lib/src/markdown_processor.dart [hunk 3]
```diff
@@ -647,6 +647,10 @@ class _MarkdownCommentReference {
 
     for (Class tryClass in tryClasses) {
       if (tryClass != null) {
+        if (codeRefChomped.contains('.') &&
```
Comment (devoncarew) [inline lib/src/markdown_processor.dart]: Could you add a brief line comment describing what this is doing.

File: test/model_test.dart (hunk 1 above)
Comment (devoncarew) [reply test/model_test.dart]: and, `implementor`

Comment (keertip) [reply test/model_test.dart]: fixed

File: testing/test_package/lib/override_class.dart (hunk 2 above)
Comment (keertip) [reply testing/test_package/lib/override_class.dart]: yes, formatted.

**Pull request merged**